x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/indicator_match/threat_mapping/create_threat_signals.ts

```diff
@@ -169,14 +171,83 @@ export const createThreatSignals = async ({
     let list = await getDocumentList({ searchAfter: undefined });
     let documentCount = totalDocumentCount;
 
+    // this is re-assigned depending on max clause count errors
+    let chunkPage = itemsPerSearch;
+
     while (list.hits.hits.length !== 0) {
       verifyExecutionCanProceed();
-      const chunks = chunk(itemsPerSearch, list.hits.hits);
+      const chunks = chunk(chunkPage, list.hits.hits);
       ruleExecutionLogger.debug(`${chunks.length} concurrent indicator searches are starting.`);
       const concurrentSearchesPerformed =
         chunks.map<Promise<SearchAfterAndBulkCreateReturnType>>(createSignal);
+
       const searchesPerformed = await Promise.all(concurrentSearchesPerformed);
-      results = combineConcurrentResults(results, searchesPerformed);
+
+      // Did our searches fail with an error containing the maxClauseCount
+      // error message?
+
+      // TODO - delete this comment
+      // FOR PR TESTING PURPOSES ONLY
+      // uncomment this line..
+      // const maxClauseCountValue = MAX_CLAUSE_DEFAULT_VALUE;
```

Comment:
should these comments be deleted?

Reply:
Just kept these in here as a reference point from the PR description for people who test the PR. I'll delete them before merging 👍